### DISCUSSION TITLE
[Importer] Define multiple IMPORT_DIR_WHITELIST folders

### DISCUSSION THREAD

**Discussion by stphn17** [firefly-iii/firefly-iii#6746] [Firefly III] [answered]
Hello, 
I am currently setting up Firefly and the Importer for the first time and I was able to set everything up following the official documentation and some trial and error. However, I have troubles setting up multiple folders for imports, or is this not at all possible?

A white **list** implies that there can be multiple values.
Even [the code](https://github.com/firefly-iii/data-importer/blob/de87fe01c87dea2cf329e5cfca8ba4803e283023/app/Console/HaveAccess.php#L70) seems to imply that there are mutliple values possible, but whatever I tried, it's not.
I am using docker compose and I tried following things:
1. defining the variable IMPORT_DIR_WHITELIST two times with different values
2. seperating the folders with space
3. seperating the folders with comma

But all unsuccessful.
I always get the following output:
```
Docker Compose:
- IMPORT_DIR_WHITELIST=/var/www/html/storage/uploads /var/www/html/storage/configurations

Bash:
ash-4.4# docker exec -it firefly_data-importer php artisan importer:auto-import storage/configurations
Path "/var/www/html/storage/configurations" is not in the list of allowed paths (IMPORT_DIR_WHITELIST).
```

Hope you can help. Thank you!

**Comment by JC5** [comment, marked as answer]
The code uses comma's to separate paths. I don't see why it shouldn't work, but keep in mind that Docker may need you to add "quotes" around complex variables.

**Reply by stphn17** [reply]
I tried with commas before, but didn't work. I have now tried again **but without a space after the comma** and it's working now. I feel stupid.
`- IMPORT_DIR_WHITELIST=/var/www/html/storage/configurations,/var/www/html/storage/uploads`
works for both folders now.

Thank you!
